src/main/java/com/jcabi/s3/Bucket.java

```diff
@@ -120,7 +120,10 @@ void remove(@NotNull(message = "S3 key can't be NULL") String key)
          */
         private final transient Bucket origin;
         /**
-         * Prefix.
+         * Prefix is a string which is used.
```

Comment:
Someone who is reading this on a Javadoc page will not be able to see the source code directly. Better to show a short example right here.